release 1.0.0

jasontwong opened the pull request:
- no longer support v1 api keys/user tokens
- no longer uses organization name

lee-reinhardt commented:
I switched to using the sdk release/1.0.0 branch and am getting these compilation errors:
<img width="748" alt="Screenshot 2025-05-23 at 11 24 00 AM" src="https://github.com/user-attachments/assets/20cdbece-b062-4cdc-851c-fd1d83ca80a4" />

jasontwong commented:
> I switched to using the sdk release/1.0.0 branch and am getting these compilation errors: <img alt="Screenshot 2025-05-23 at 11 24 00 AM" width="748" src="https://private-user-images.githubusercontent.com/980089/447117152-20cdbece-b062-4cdc-851c-fd1d83ca80a4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDgwMjUzODUsIm5iZiI6MTc0ODAyNTA4NSwicGF0aCI6Ii85ODAwODkvNDQ3MTE3MTUyLTIwY2RiZWNlLWIwNjItNGNkYy04NTFjLWZkMWQ4M2NhODBhNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUyM1QxODMxMjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZTAxZDUzNzFlZDJlNGNkMGNmN2M2M2Q5MTc4NmRmODJkZGJmYmVjZDM5MGNkM2MyYTRmNGJhNWRhZjlkOWFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aiBWU3_VKsfjEbzqxeN3HNP_zK9UCvn9W3CAPx8VTa4">

These are specific to the CLI which I have already fixed.
